config/config.go

```diff
@@ -55,6 +60,7 @@ type Config struct {
 	Zookeeper   *zookeeper.Config `yaml:"zookeeper"`
 	Admin       AdminConfig       `yaml:"admin"`
 	Controller  *ControllerConfig `yaml:"controller"`
+	ApiAuth     *ApiAuthConfig    `yaml:"apiAuth"`
```

Comment:
I see that we we snake_case for yaml-tags in this file. Could you please adhere this convention?